File: Selenium/Project/Activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.core import driver
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service=service) as driver:
    driver.get("https://alchemy.hguy.co/jobs")
    driver.find_element(By.XPATH,"//a[@href='https://alchemy.hguy.co/jobs/post-a-job/']").click()
    abc=driver.title
    print(" page title: " + abc)
    driver.find_element(By.XPATH,"//input[@id='create_account_email']").send_keys("12334@gmail.com")
    driver.find_element(By.XPATH,"//input[@id='job_title']").send_keys("ApplicationTester")
    driver.find_element(By.XPATH,"//input[@id='job_location']").send_keys("Pune")
    driver.find_element(By.XPATH,"//iframe[@id='job_description_ifr']").send_keys("Applicantes have knowledge on mediaton appliaction and have minimum 2 years experience")
    driver.find_element(By.XPATH,"//input[@id='application']").send_keys("uchallag@in.ibm.com")
    driver.find_element(By.XPATH,"//input[@id='company_name']").send_keys("IBM")
    driver.find_element(By.XPATH,"//input[@value='Preview']").click()
    driver.find_element(By.XPATH,"//input[@value='Submit Listing']").click()
    driver.find_element(By.XPATH,"//a[@href='https://alchemy.hguy.co/jobs/jobs/']").click()
    driver.find_element(By.id,"search_keywords").send_keys("ApplicationTester")
    driver.find_element(By.XPATH,"//input[@value='Search Jobs']").click()
    driver.quit()

File: Selenium/Project/Activity7.java
package SelJava;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity7
{
    WebDriver driver;

    @BeforeClass
    public void beforeMethod()
    {
        WebDriverManager.firefoxdriver().setup();
        driver = new FirefoxDriver();

        driver.get("https://alchemy.hguy.co/jobs");
    }

    @Test
    public void exampleTestCase()
    {
        driver.findElement(By.xpath("//a[@href='https://alchemy.hguy.co/jobs/post-a-job/']")).click();

        driver.findElement(By.xpath("//input[@id='create_account_email']")).sendKeys("tsaiprasad18@gmail.com");
        driver.findElement(By.xpath("//input[@id='job_title']")).sendKeys("Manual QA");
        driver.findElement(By.xpath("//input[@id='job_location']")).sendKeys("Hyderabad");
        //driver.findElement(By.xpath("//input[@id='job_type']")).sendKeys("Hybrid");
        driver.findElement(By.xpath("//iframe[@id='job_description_ifr']")).sendKeys("  Hi.......Bye");
        driver.findElement(By.xpath("//input[@id='application']")).sendKeys("tsaiprasad18@gmail.com");
        driver.findElement(By.xpath("//input[@id='company_name']")).sendKeys("IBM INDIA PVT. LTD.");
        driver.findElement(By.xpath("//input[@id='company_website']")).sendKeys("http://ibm.com");
        driver.findElement(By.xpath("//input[@id='company_tagline']")).sendKeys("IBM WatsonX Welcomes You");
        driver.findElement(By.xpath("//input[@id='company_video']")).sendKeys("video.mp4");
        driver.findElement(By.xpath("//input[@id='company_twitter']")).sendKeys("tsaiprasad18");

        driver.findElement(By.xpath("//input[@value='Preview']")).click();

        driver.findElement(By.xpath("//input[@value='Submit Listing']")).click();

        driver.findElement(By.xpath("//a[@href='https://alchemy.hguy.co/jobs/jobs/']")).click();
        driver.findElement(By.id("search_keywords")).sendKeys("Manual QA");
        driver.findElement(By.xpath("//input[@value='Search Jobs']")).click();

    }

    @AfterClass
    public void afterMethod()
    {
        driver.quit();
    }
}